cpp/examples/sbml2memilio.cpp

```diff
@@ -0,0 +1,53 @@
+#include "sbml2memilio.h"
```

Comment:
we already have a lot of examples. Im against splitting this in 2 files.  As this is not part of the libary, this is fine in my opinion.
Also the naming is not alligning with our notation. Use underscores :)